gammapy/data/observations.py

```diff
@@ -557,7 +557,9 @@ def read(cls, event_file, irf_file=None):
             **irf_dict,
         )
 
-    def write(self, path, overwrite=False, format="gadf", include_irfs=True):
+    def write(
+        self, path, overwrite=False, format="gadf", include_irfs=True, checksum=False
```

Comment:
Can you add in the docstring also the default value of the parameters?

Reply:
done